vyxal/elements.py

```diff
@@ -6195,6 +6195,23 @@ def connected_uniquify(lhs, ctx: Context):
     }.get(ts)()
 
 
+def multidimensional_truthy_indices(lhs, ctx: Context):
```

Comment:
By the way, there's a function called `enumerate_md` or something to enumerate a multidimensional list. It could help shorten this (sorry for not mentioning earlier)

Reply:
I know it exists, but I was too lazy to go figure out how on earth you use it.